app/root.tsx

```
@@ -13,16 +15,28 @@ import ErrorPage from 'src/components/error-page'
+
   return (
-    <html lang="en">
+    <html lang={i18n.language} dir={i18n.dir()}>
       <head>
         <meta charSet="utf-8" />
         <link rel="manifest" href="/manifest.json" />
 
         <Meta />
         <Links />
 
+        <script dangerouslySetInnerHTML={{ __html: `window.ENV = ${JSON.stringify(clientEnv)}` }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)